src/System.Management.Automation/engine/regex.cs

```diff
@@ -58,7 +58,10 @@ public sealed class WildcardPattern
         private const int StackAllocThreshold = 256;
 
         // chars that are considered special in a wildcard pattern
-        private const string SpecialChars = "*?[]`";
+        private static readonly SearchValues<char> SpecialChars = SearchValues.Create("*?[]`");
```

Comment:
Happy to not include this if it's not relevant for this PR. It does however work just fine in `Pattern.AsSpan().IndexOfAny(SpecialChars)`.

It just makes sense symmetry wise to keep them both `SearchValues<char>`

Reply:
I am also happy to just take the SearchValues<char> conversion if its not needed, I think fix is to just check backtick separately.

Reply:
This makes sense since I don't think OR can be outperformed. In any case I have reverted and kept changes small for the bug fix.